storages/python/python-storage-plugins/redis.py

```diff
@@ -44,7 +47,11 @@ def open(self, location: str, options=None):
 
         # Fix option types and open Redis connection
         opts.port = int(opts.port)
-        opts.ssl = dlite.asbool(opts.ssl)
+        opts.socket_keepalive = dlite.asbool(opts.socket_keepalive)
+        if "socket_timeout" in opts:
```

Comment:
Do you have to supply the `socket_timeout` parameter in order for redit not to hang? In that case, a typical working value would be helpful in the parameter description.

Do you know if this value is used by the keep-alive functionality, or if `socket_timeout` only configures other behavior?

Reply:
Again, adding `socket_timeout` was a part of my testing for a solution for the hanging. It turned out not to help. I kept it as an option for the plugin in case a user of DLite wants to play with. The default is left to redis. 
I think that a good working value depends on the situation - redis is fast, so I guess it could be a lot less than a second.

Reply:
Thank you for the clarification!